doc/kernel_tests.rst

```diff
@@ -21,6 +21,53 @@ In our case, the data usually consists of
 `Ftrace <https://www.kernel.org/doc/Documentation/trace/ftrace.txt>`_ traces
 that we then postprocess using :mod:`trappy`.
 
+Available tests
+===============
+
+The following tests are available. They can be used as:
+  * direct execution using ``lisa-test`` command (``LISA shell``) and ``exekall``
+  * the individual classes/methods they are composed of can be used in custom
+    scripts/jupyter notebooks (see ipynb/tests/synthetics_example.ipynb)
+
+.. include:: test_list.rst
+
+Scheduler tests
```

Comment:
I'd like to keep the "generic" bits before the API description - the "Writing tests" section is sandwiched between the tests implementation and the base API.

I'd rather see something like this:

```
* Introduction
* Available tests
* Writing tests
* API
** Base API
** Scheduler tests
** blah blah
```

Reply:
sounds good, the new version has that order. The test list acts as an index at the beginning, which allows us to have a more logical order for the actual descriptions.